uwblueprint/project-read-frontend
# Signs out user during the first request they make of every day

(cant find the ticket)



### Implementation description

- Checks the date last logged in stored in the ID token and if it isn't today, signs them out



### Steps to test

1. Login
2. Wait a day
3. Revisit the site, and it should require you to log in again
